src/client/Client.cc

```diff
@@ -1336,6 +1337,31 @@ void Client::insert_readdir_results(MetaRequest *request, MetaSession *session,
   }
 }
 
+void Client::parse_create_reply_extra(MetaRequest *request, MetaSession *session)
+{
+  auto p = request->reply->get_extra_bl().cbegin();
+  if (!p.end()) {
+    if (session->mds_features.test(CEPHFS_FEATURE_DELEG_INO)) {
+      struct openc_response_t ocres;
+
+      decode(ocres, p);
+      request->created_ino = ocres.created_ino;
+      /*
+       * The userland cephfs client doesn't have a way to do an async create
+       * (yet), so just discard delegated_inos for now. Eventually we should
+       * store them and use them in create calls, even if they are synchronous,
+       * if only for testing purposes.
+       */
```

Comment:
Isn't the above comment no longer accurate with this patch? It should probably just be removed.